CONTRIBUTING.md

```diff
@@ -0,0 +1,179 @@
+# Contributing to the Tonic site
+
+Thank you for taking the time to contribute to the Tonic site.
+Contributions are welcome and this document is intended to help them to find their way into the project.
```

Comment:
Hm, I get your point, but it sounds quite defensive to me. I'll have to think this over a bit.

Reply:
I've added most of your suggestion because I thought of a cultural difference between scientific and open source communities. In open source projects, it is much more natural that everyone is invited to participate, whereas in science, this is no matter of course and a more explicit invitation might be appropriate. I have left out the more apologizing parts, because I think apologizing for length doesn't make it shorter. ;-)  And ironically, if I had targeted only advanced users, the guide would have become remarkably shorter.